test/config/ocsp-updater.json

```diff
@@ -8,8 +8,8 @@
     "revokedCertificateWindow": "1s",
     "newCertificateBatchSize": 1000,
     "oldOCSPBatchSize": 5000,
-    "parallelGenerateOCSPRequests": 10,
     "missingSCTBatchSize": 5000,
+    "parallelGenerateOCSPRequests": 10,
```

Comment:
Yep, I mentioned it in the desc. This makes the ordering identical between config and config-next so diffs are minimized.